Sources/Core/Request/Request+EmptyDecodingStrategy.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  Request+EmptyDecodingStrategy.swift
+//  Hyperspace
+//
+//  Copyright © 2020 Bottle Rocket Studios. All rights reserved.
+//
+
+import Foundation
+
+// MARK: - EmptyResponse
+
+// MARK: - Empty Response Request Default Implementations
+public extension Request where Response == Void {
+
+    static func withEmptyResponse(method: HTTP.Method = .get,
+                                  url: URL,
+                                  headers: [HTTP.HeaderKey: HTTP.HeaderValue]? = nil,
```

Comment:
I'm sure you thought about this thoroughly, but: prefer empty headers instead of null? I don't like null arrays if we can use an empty array instead.

Reply:
That's a fair criticism. The only real reason I used the optional dictionary here was to match the `Request` initializer (which I believe was modeled after `URLRequest`, which uses `[String: String]?`)

I don't actually know if it generates any concrete difference in the request though whether it's `nil` or `[:]`.